src/js/utils/SetupUtil.js

```diff
@@ -28,6 +28,8 @@ const SetupUtil = {
         preparedData[key] = SetupUtil.getArrayFromHostsString(newFormData[key]);
       } else if (key === 'ssh_port') {
         preparedData[key] = parseInt(newFormData[key]);
+      } else if (key === 'nested_cluster_options') {
```

Comment:
I'm curious what you mean by `nested_cluster_options` it doesn't mean anything to me. So I'm confused by the name.